### PR TITLE
Using evaluateJavascript to fix #9749

### LINKED ISSUE #9749: Comments on WebView injected script will break the scipt
### Issue Description

When injecting a script to a `WebView` component via `injectedJavaScript` prop, if the script contains any comment then it won't work on android.
### Steps to Reproduce / Code Snippets

Create a `WebView` in which we inject a script with a simple comment in it.
#### Code to reproduce

``` es6
import React from 'react';
import { AppRegistry, StyleSheet, Text, View, WebView } from 'react-native';

function SampleApp () {
  const html = `
    <html>
      <body>
        <h1 class="title"> Hello! </h1>
      </body>
    </html>
  `;
  const script = `
    var text = "Bye!"; // This comment will break the script on android
    document.querySelector(".title").textContent = text;
  `;
  return (
    <View style={styles.container}>
      <Text> WebView script comment issue </Text>
      <WebView
        source={{html: html}}
        injectedJavaScript={script}
        javaScriptEnabled
      />  
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'stretch',
    backgroundColor: '#F5FCFF',
    paddingTop: 50
  }
});

AppRegistry.registerComponent('SampleApp', () => SampleApp);
```

[Working example](https://rnplay.org/apps/TWZWYA)
#### Expected Results

Text on html `h1` element ("Hello!") should be replaced with "Bye!" text.
#### Actual Results

Text on `h1` html element changes on iOS but not on Android. Removing the comment on the js script will fix this issue.
### Additional Information
- React Native version: 0.31.0
- Platform(s) (iOS, Android, or both?): both
- Operating System (macOS, Linux, or Windows?): macOS

### PR DESCRIPTION
**Pull request by leeight**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

Use `evaluateJavascript` instead of `loadUrl` if possible, which will preserve the single line comment.

**Test plan (required)**

Manually.

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @sathyapriya-31 and @jvassbo to be potential reviewers.

**Comment by leeight** [comment]
Ping @mkonicek

**Comment by satya164** [comment]
Doesn't it mean that the behaviour becomes inconsistent across Android versions? People might test on new Android versions and not realize that the same code will break on older Android versions.

**Comment by leeight** [comment]
@satya164 
image

**Comment by jacobp100** [comment]
LGTM. There will obviously be some edge cases, where the ‘//‘ is in a string, but that probably is not worth trying to resolve.

File: Libraries/Components/WebView/WebView.android.js [hunk 1]
```diff
@@ -207,6 +207,14 @@ class WebView extends React.Component {
       console.warn('WebView: `source.body` is not supported when using GET.');
     }
 
+    if (__DEV__) {
+      if (this.props.injectedJavaScript
+          && this.props.injectedJavaScript.indexOf('//') !== -1) {
```
Comment (satya164) [inline Libraries/Components/WebView/WebView.android.js]: This will also cause warning if any string matches this

**Comment by leeight** [comment]
OK, Close it.

**Comment by LinusU** [comment]
#20366 should fix this in a way that behaves the same on all versions of Android

**Pull request closed without merging**